src/search.c

```diff
@@ -309,7 +309,7 @@ static GtkWidget *add_find_checkboxes(GtkDialog *dialog)
 	ui_hookup_widget(dialog, check_regexp, "check_regexp");
 	gtk_button_set_focus_on_click(GTK_BUTTON(check_regexp), FALSE);
 	gtk_widget_set_tooltip_text(check_regexp, _("Use POSIX-like regular expressions. "
-		"For detailed information about using regular expressions, please read the documentation."));
+		"For tips about using regular expressions, please read the Geany manual."));
```

Comment:
Nah, at worse it refers to "regular expressions", but then wouldn't you use "their documentation"?

I guess it's the same in French, where yes, it's not strictly speaking unambiguous, but it's perfectly clear in the context.

---
and even if it wasn't, it should be perfectly fine for someone to look up "PCRE docs", whatever that would be.